src/cbmc/bmc.cpp

```diff
@@ -69,6 +69,9 @@ Function: bmct::error_trace
 
 void bmct::error_trace()
 {
+  if(options.get_bool_option("stop-when-unsat"))
+    return;
+
```

Comment:
The following regression tests use "stop-when-unsat":
cbmc-incr/induction1
cbmc-incr-oneloop/induction1
cbmc-incr-oneloop/induction2